### PR TITLE
Ngstack 941 autocomplete

### PR DESCRIPTION
**Pull request by zulicek**

### CODE REVIEW

File: translations/messages.en.yaml [hunk 1]
```diff
@@ -59,4 +62,4 @@ ngsite.share_on_facebook: 'Share on Facebook'
 
 ngsite.change_language: 'Change language'
 
-ngsite.open_navigation: 'Open main navigation'
+ngsite.open_navigation: 'Open main navigation'
```
Comment (emodric) [inline translations/messages.en.yaml:65]: Missing new line at EOF

File: translations/messages.de.yaml [hunk 2]
```diff
@@ -30,6 +30,9 @@ ngsite.search.no_results.reduce_keywords: 'Reduzieren Sie die Anzahl der Keyword
 ngsite.collected_info.success_text: 'Wir danken Ihnen für Ihr Feedback.'
 ngsite.collected_info.button.send: 'Formular senden'
 ngsite.collected_info.information_collected: 'Die folgenden Informationen wurden gesammelt'
+ngsite.collected_info.given_name.attribute: 'given-name'
```
Comment (emodric) [inline translations/messages.de.yaml:33]: Why is this in translation?

File: templates/themes/app/content/views/payload/ng_contact_form.html.twig [hunk 3]
```diff
@@ -15,14 +15,14 @@
         {{ form_errors(form) }}
 
         <fieldset>
-            {{ form_row(form.sender_first_name) }}
+            {{ form_row(form.sender_first_name, { ngparams: { autocomplete: 'ngsite.collected_info.given_name.attribute'|trans } })}}
```
Comment (emodric) [inline templates/themes/app/content/views/payload/ng_contact_form.html.twig:18]: I believe you don't need to use `ngparams`. It looks like `ngparams` was used to override already existing `label` value with our own implementation (for which I'm not 100% clear why, btw). This should work by only transfering `autocomplete` variable.

File: translations/messages.de.yaml (hunk 2 above)
Comment (zulicek) [reply translations/messages.de.yaml:33]: these are predefined autocomplete tokens

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

should I just write them in the form?
`{{ form_row(form.sender_first_name, { ngparams: { autocomplete: 'given-name' } })}}`

File: templates/themes/app/content/views/payload/ng_contact_form.html.twig (hunk 3 above)
Comment (zulicek) [reply templates/themes/app/content/views/payload/ng_contact_form.html.twig:18]: it doesn't work when I change it to` {{ form_row(form.sender_first_name, { autocomplete: 'ngsite.collected_info.given_name.attribute'|trans } })}`

File: translations/messages.en.yaml (hunk 1 above)
Comment (zulicek) [reply translations/messages.en.yaml]: corrected

File: templates/themes/app/content/views/payload/ng_contact_form.html.twig (hunk 3 above)
Comment (emodric) [reply templates/themes/app/content/views/payload/ng_contact_form.html.twig]: Alright then :+1:

**Review by emodric — APPROVED** [review]

**Pull request merged**